### PR TITLE
[MM-52211] Fix mono-repo issue in test workflow

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
This fixes the test worklfow that was not being modified after the mono-repo. Currently we are using the latest pre-monorepo release so that we can continue running the tests.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52211

### CODE REVIEW

**Review by agnivade — APPROVED** [review]
@mgdelacroix - is the mmctl integration within monorepo still being worked on?

**Comment by isacikgoz** [comment]
Apparentyl I picked up a very wrong branch name :p

**Review by agarciamontoro — APPROVED** [review]

**Comment by mgdelacroix** [comment]
@agnivade yes, we're finishing the required updates to the release pipeline and should be merging soon

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -69,10 +70,10 @@ jobs:
...
         go-version: 1.18.x
 
-    - name: Setup mattermost-server
-      working-directory: mattermost-server
-      run: |
-        git checkout $GITHUB_HEAD_REF || git checkout master
+   # - name: Setup mattermost-server
+   #   working-directory: mattermost-server
+   #   run: |
+   #     git checkout $GITHUB_HEAD_REF || git checkout master
```
[the remark this answers is not in the document]
Comment (isacikgoz) [reply .github/workflows/test.yml]: Discussed offline, I submitted a fix for that.

**Pull request merged**